objectsapiclient/models.py

```diff
@@ -43,9 +42,11 @@ class Meta:
     def __str__(self):
         return "Objects API client configuration"
 
-    @cached_property
     def client(self):
```

Comment:
Made a mistake earlier, should be:

```suggestion
    @property
    def client(self):
```

Reply:
Deleted without checking :see_no_evil: